common/arrow/Cargo.toml

```diff
@@ -15,7 +15,7 @@ simd = ["arrow/simd"]
 # Workspace dependencies
 
 # Github dependencies
-arrow = { package = "arrow2", git="https://github.com/datafuse-extras/arrow2", rev = "647fddb" }
```

Comment:
It's not enough because the high-version library isn't compatible with old-version.
Need more modification.